content/logs/aws.md

```diff
@@ -47,10 +47,37 @@ Push your AWS log information to Datadog using a Lambda function bound to an S3
 
 ## Collection
 
-Your lambda function is now ready to send logs to your Datadog platform.
-Setup the relevant triggers for each AWS service you want to monitor.
-
-### S3, Cloudwatch, API Gateway, Kinesis and SNS
+Your lambda function is now ready to send logs to your Datadog platform. Setup the relevant triggers for each AWS service you want to monitor.
+
+**The Lambda function you just created must be in the same region as the S3 bucket you are using as a trigger. If you have logs across multiple regions you must create additional Lambda funcions**
```

Comment:
here you refer to it as a trigger (correct) but above in the overview its a binding.